test/e2e/case4_unexpected_policy_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"open-cluster-management.io/governance-policy-propagator/test/utils"
 )
 
-var _ = Describe("Test unexpect policy handling", func() {
+var _ = Describe("Test unexpect policy handling", Label("non-placement-rule"), func() {
```

Comment:
Everything is good! but  Why is this test in non-placementRule?

Reply:
This is the one test I found that didn't use PlacementRule. In the future, I'm hoping we have others (maybe we already do, I didn't look super hard).

For this PR, I just needed at least one test for that suite to make sure that the controller didn't crash when the PlacementRule CRD was no longer present.